src/testdir/test_functions.vim

```diff
@@ -749,6 +749,7 @@ endfunc
 
 " Test for the mode() function
 func Test_mode()
+  CheckScreendump
```

Comment:
Most part of this test doesn't require screendump. It's better to move the part that uses screendump to a separate test, so that this test is still run for FEAT_NORMAL builds.